app.js

```diff
@@ -1,13 +1,14 @@
 const express = require('express')
 
-const {getCat, getRev, getRevById} = require('./controller.js')
+const {getCat, getReview, getRevById, getComFromId} = require('./controller.js')
```

Comment:
It would be really useful to keep the naming consistent here so anyone not familior with the project (or when you re-visit the code in a few weeks time) knows what everything does. 

I wouldn't recommend shortening the function names, try to keep them as descriptive as possible.